Pod/Classes/GDGEntity.h

```diff
@@ -13,21 +12,21 @@
 
 @protocol GDGEntityFillDelegate <NSObject>
 
-- (void)entity:(GDGEntity*)entity hasFilledPropertyNamed:(NSString*)propertyName;
-- (void)entity:(GDGEntity*)entity requestToFillPropertyNamed:(NSString*)propertyName;
+- (void)entity:(GDGEntity *)entity hasFilledPropertyNamed:(NSString *)propertyName;
+
+- (void)entity:(GDGEntity *)entity requestToFillPropertyNamed:(NSString *)propertyName;
 
 @end
 
 @interface GDGEntity : NSObject
 
 @property (assign, nonatomic) NSUInteger id;
-@property (readonly, nonatomic) GDGEntityManager<GDGEntityFillDelegate>* db;
+@property (readonly, nonatomic) GDGEntityManager <GDGEntityFillDelegate> *db;
 
-- (BOOL)save;
-- (BOOL)drop;
++ (instancetype)entity;
```

Comment:
Meio estranho `[GDGEntity entity];` não? Sei que não é Java, então nada de `newInstance`, mas se for pensar em `sharedManager` ou `defaultManager`, `newEntity` não parece tão ruim.